File: src/main/java/no/fintlabs/dispatch/model/InstanceReceiptDispatchEntity.java
package no.fintlabs.dispatch.model;

import lombok.*;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class InstanceReceiptDispatchEntity {
    @Id
    private String sourceApplicationInstanceId;
    private String uri;
    private Class<?> classType;
    @Column(columnDefinition = "TEXT")
    private String instanceReceipt;
}

File: src/main/kotlin/no/novari/flyt/egrunnerverv/gateway/infrastructure/dispatch/model/InstanceReceiptDispatchEntity.kt
package no.novari.flyt.egrunnerverv.gateway.infrastructure.dispatch.model

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "instance_receipt_dispatch_entity")
class InstanceReceiptDispatchEntity(
    @Id
    @Column(name = "source_application_instance_id")
    var sourceApplicationInstanceId: String = "",
    @Column(name = "uri")
    var uri: String = "",
    @Column(name = "instance_receipt", columnDefinition = "TEXT")
    var instanceReceipt: String = "",
)
